libminifi/test/pcap-tests/PcapTest.cpp

```diff
@@ -69,9 +69,10 @@ class PcapTestHarness : public IntegrationBase {
   void runAssertions() {
     using org::apache::nifi::minifi::utils::verifyLogLinePresenceInPollTime;
     assert(verifyLogLinePresenceInPollTime(std::chrono::milliseconds(wait_time_),
-        "Starting capture",
-        "Stopping capture",
-        "Stopped device capture. clearing queues",
+        // These assertions don't work, but the test is still useful to check that the processor starts
```

Comment:
Maybe we could add a TODO or a FIXME message here

Reply:
done